src/main/scala/no/ndla/articleapi/service/converters/HTMLCleaner.scala

```diff
@@ -73,4 +72,29 @@ object HTMLCleaner extends ConverterModule {
     el.html(el.html().replace("\u00a0", "")) // \u00a0 is the unicode representation of &nbsp;
   }
 
+  def isAttributeKeyValid(attributeKey: String, tagName: String): Boolean = {
+    val legalAttributesForAll = Set("href", "title")
```

Comment:
Noen grunn til at du setter opp mapene for hvert kall til funksjonen?
Dersom det er struktur som ikke er generelt interessant for hele klassen ville jeg heller gruppert ved å opprette et internt objekt som holder denne, slik at ikke statisk initialisering blandes med logikken.